server/views/pages/case/select.njk

```diff
@@ -29,9 +29,9 @@
 
     {{ supportBox() }}
 
-  {% if user.userinfo %}
+  {% if user %}
```

Comment:
I would undo this change. Outside of scope. But we should add this once everything is merged